docs/development/decision-records/2024-09-16-tx-target-node-directory-proposal/README.md

```diff
@@ -0,0 +1,66 @@
+> The Federated Catalog requires a list of Target Catalog Nodes, so it knows which endpoints to crawl. This list is provided by the TargetNodeDirectory. During the preparation of a crawl run, the ExecutionManager queries that directory and obtains the list of TCNs.
+
+So having a service containing the data of desired BPNL's and respective Connectors' URL's that a certain partner wants, but here the partner itself has the service in their scope and users are able to input (in the API of new service) the Participants.
+
+This solution allows the member to choose the relevant data that interests them (offers from specific participants) enabling the reduction of storage cost and less network calls. Additionally, each member has control on how to host and manage this new service, service changes do not affect other parties (unless contract changes) and is able to be independently scalable.
+
+
+Other solutions were also considered
+
+- File in a S3 bucket (or different cloud provider's solution)
+    - This solution was discarded due to One file for all instead of each partner having the data that respectively needs does not match the requirement and this solution would lock the usage of a proprietary tool (cloud provider) being harder to sustain in an open source community.
+
+- Centralized service with all participants data
+    - Solution was not considered due to presenting a single point of failure, the partner still needs to have a way to dynamically store on desired partners ids' and each partner must have the BPNL or the Connectors' URL's beforehand.
+
+## Approach
+
+Using the Discovery Service, the FC is able to request the connetors' url's that it requires, providing the BPNL's. Since the Discovery Service has already built in this feature, the work of this flow is already relatively well established.
+However, the main question here arises: what if a new partner joins the space? The existing partners that want to obtain the offers of the new partner do not know of its existence. So an additional mechanism is needed. The default implementation is a static file, however a dynamic approach would be preferable (no downtime expected).
+
+Even with a new service registering the Discovery Service (basically making it available in the space to other partners) the existing partners that may want to retrieve the new partners' offers must establish the need to retrieve them by defining the new BPNL somewhere.
+
+
+A service containing the data of desired BPNL's and respective Connectors' URL's that a certain partner wants, but here the partner itself has the service in their scope and users are able to input (in the API of new service) the Participants. Simply, the new service would have a db/cache containing the BPNL's and Connectors' URL's of each partner this member wants the offers from. Again, **this new service works as the TargetNodeDirectory for the partner's FederatedCatalog**.
```

Comment:
If you feel the need to put something in simpler terms, why do you keep both forms and not only the simpler one?

Reply:
Yeah. Removed duplicated content in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1556/commits/fae5e8bafbc033e16bcfce259fc95cb2ad7b06af).